server/lomas_server/tests/test_data/test_datasets_with_s3.yaml

```diff
@@ -13,15 +13,15 @@ datasets:
     metadata_path: "tests/test_data/metadata/penguin_metadata.yaml"
 - dataset_name: "TINTIN_S3_TEST"
   database_type: "S3_DB"
-  s3_bucket: "example"
-  s3_key: "data/test_penguin.csv"
+  bucket: "example"
+  key: "data/test_penguin.csv"
   endpoint_url: "http://localhost:9000"
-  aws_access_key_id: "admin"
-  aws_secret_access_key: "admin123"
+  credentials_name: "local_minio"
   metadata:
     database_type: "S3_DB"
-    s3_bucket: "example"
-    s3_key: "metadata/penguin_metadata.yaml"
+    bucket: "example"
+    key: "metadata/penguin_metadata.yaml"
     endpoint_url: "http://localhost:9000"
-    aws_access_key_id: "admin"
-    aws_secret_access_key: "admin123"
\ No newline at end of file
+    access_key_id: "admin"
```

Comment:
We still need credentials in a yaml file here. This is for the mongo_admin_cli tool to be able to fetch the metadata since it does not have access to the server config.

One solution could be to add endpoints to the server for these admin tasks. This would probably enhance the security and modularity aspects.